packages/block-library/src/gallery/style.scss

```diff
@@ -65,6 +65,10 @@
 		}
 	}
 
+	figcaption {
```

Comment:
I think this can actually go in the existing `figcaption` ruleset above, right? Applied to `.blocks-gallery-image` and `.blocks-gallery-item`?

Reply:
No, It should not go inside `.blocks-gallery-image` and `.blocks-gallery-item` class's `figcaption`. Because that will be applied only for a particular image's figcaptions.

Reply:
Ah yes of course. 😄 Thank you!